src/ServiceControl.Monitoring/Infrastructure/BreakdownRegistry.cs

```diff
@@ -37,7 +37,7 @@ public void Record(BreakdownT breakdown)
 
         public IEnumerable<BreakdownT> GetForEndpointName(string endpointName)
         {
-            return lookup[endpointName];
+            return lookup.ContainsKey(endpointName) ? lookup[endpointName] : new List<BreakdownT>();
```

Comment:
This is a concurrently accessed field. Also, the lookup can be replaced by `TryGet`. To address these two:

```
if (lookup.TryGet(endpointName, out breakdowns))
...
```